utils/rpms/daos.spec

```
@@ -41,7 +41,7 @@ BuildRequires: hwloc-devel
 BuildRequires: bullseye
 %endif
 %if (0%{?rhel} >= 8)
-BuildRequires: argobots-devel >= 1.1
+BuildRequires: argobots-devel >= 1.2
```

You also need to update https://github.com/daos-stack/daos/blob/861445b7ff2d49880a4df4f71f4e5ddd6279208e/utils/rpms/daos.spec#L48

But probably a better way to handle this is to add a new 
```
%global argobots_version 1.2
```

after https://github.com/daos-stack/daos/blob/861445b7ff2d49880a4df4f71f4e5ddd6279208e/utils/rpms/daos.spec#L7 and then you can do this here:
```suggestion
BuildRequires: argobots-devel >= %{argobots_version}
```

and then

```patch
-BuildRequires: libabt-devel >= 1.0rc1
+BuildRequires: libabt-devel >= %{argobots_version}
```

on line 48.

---

- [ ] Improve RPM build dep version handling